inginious/frontend/templates/course_admin/submissions.html

```diff
@@ -143,15 +143,15 @@ <h2>{{_("Submissions")}}</h2>
                 {% if "show_tags" in old_params %}
```

Comment:
If we only use the category concept in this page, it's worth renaming tags by categories.

Reply:
I think here, the concept of tags is still the one used